### PR TITLE
GLSP-1399: Fix MousePositionTracker

### PR DESCRIPTION
**Pull request by tortmayr**
#### What it does

Ensure that the `GLSPMousePositionTracker` correctly calculates the diagram local mouse position

#### How to test

Ensure that resize/move still works as expected.
Also test the corner case with resizing/moving near/over the projection bars
If you want to verify the tracked position you can add a log point in the `GLSPMousePositionTracker.mouseMove` method
#### Changelog

-   [x] This PR should be mentioned in the changelog
-   [ ] This PR introduces a breaking change (if yes, provide more details below for the changelog and the migration guide)

### CODE REVIEW

**Review by martin-fleck-at — APPROVED** [review]
Change looks good to me and seems to work well. Just unsure whether we want to keep the now unused field injection.

File: packages/client/src/base/mouse-position-tracker.ts [hunk 1]
```diff
@@ -38,9 +29,9 @@ export class GLSPMousePositionTracker extends MousePositionTracker implements Ra
     override mouseMove(target: GModelElement, event: MouseEvent): (Action | Promise<Action>)[] {
         // we cannot simply use the offsetX and offsetY properties of the event since they also consider nested HTML elements
         // such as foreignObjects or the projection bars divs. Instead, we manually translate the client coordinates to the diagram
-        const clientToRoot = getAbsoluteClientBounds(target.root, this.domHelper, this.viewerOptions);
```
[the remark this answers is not in the document]
Comment (tortmayr) [reply packages/client/src/base/mouse-position-tracker.ts:41]: You are absolutely right. I think we can just remove them

**Review by martin-fleck-at — APPROVED** [review]
LGTM!
